Framework/Algorithms/src/Rebin.cpp

```diff
@@ -112,6 +114,10 @@ void Rebin::init() {
   declareProperty(
       "FullBinsOnly", false,
       "Omit the final bin if it's width is smaller than the step size");
+
+  declareProperty("IgnoreBinErrors", false, "Ignore errors related to "
+                                            "zero/negative bin widths in "
+                                            "input/output workspaces.");
```

Comment:
These changes to Rebin look good and reasonable to me. 👍  Consider extending this comment to state what happens to spectra which such errors (as far as I can see Y and E are set to 0?).